hbase-client/src/main/java/org/apache/hadoop/hbase/client/MetricsConnection.java

```diff
@@ -33,14 +33,16 @@
 import java.util.concurrent.ThreadPoolExecutor;
 import java.util.concurrent.TimeUnit;
 import java.util.function.Supplier;
+
```

Comment:
Nit: extra whitespace

Reply:
We're actually going back and forth on this. I think these extra line breaks are still present in the eclipse formatter and thus are being returned via spotless. but anyway i've given up on reviewing for style because spotless is in progress.

Reply:
Thank you both. Since I got 2 approvals and spotless is imminent, I'm going to merge this and get started on the backports. Let me know if that's unacceptable @apurtell -- If i don't hear anything, i'll try to go through this whole process tomorrow morning.